Demo/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    compileSdkVersion 31
+    buildToolsVersion '30.0.3'
```

Comment:
Should buildToolsVersion also be 31?

Reply:
Ah that's a good callout - I used the Android Studio auto-upgrade feature and it put us at 30.0.3 as the latest. After looking more into it, it looks like there is a version 31 that's been released, but they should be forwards compatible (if you don't specify buildToolsVersion the default min compatible version will be used). Anyways, these build tools are used for building, debugging Android apps, command line tools, and connecting to devices, so I don't think keeping it at this version should have a merchant impact.  We can look at upgrading all the SDKs once Android 12 is GA, but for now I'd opt to keep this as is since the other SDKs have been released with this tools version.